microshift.sh

```diff
@@ -16,7 +16,7 @@ SNC_CLUSTER_CPUS=${SNC_CLUSTER_CPUS:-2}
 CRC_VM_DISK_SIZE=${CRC_VM_DISK_SIZE:-31}
 BASE_DOMAIN=${CRC_BASE_DOMAIN:-testing}
 MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp}
-OPENSHIFT_MINOR_VERSION=${OPENSHIFT_MINOR_VERSION:-4.14}
+OPENSHIFT_MINOR_VERSION=${OPENSHIFT_MINOR_VERSION:-4.16}
```

Comment:
The `release-4.15` is apparently ahead of `master`? Are we missing any commits in `master`?
Should we be getting the `release-4.16` branch into shape before having this kind of workarounds in `master`?
To be honest, our branching scheme is quite confusing :-/

Reply:
> The release-4.15 is apparently ahead of master?  Are we missing any commits in master?

you mean when you select master then it will show following? It is because we have set 4.15 as default branch.

This branch is [31 commits ahead of](https://github.com/crc-org/snc/compare/release-4.15...master), [36 commits behind](https://github.com/crc-org/snc/compare/master...release-4.15) release-4.15.


> Should we be getting the release-4.16 branch into shape before having this kind of workarounds in master?

As of now release-4.16 and master is same till the dev freeze happen on openshift side so I am putting respective changes to master and cherry-pick to release-4.16

> To be honest, our branching scheme is quite confusing :-/

I think it is now much better than before because at least now I am sure where to put respective PR and where to cherry-pick.

Reply:
> you mean when you select master then it will show following? 

I mean that this PR is changing 4.14 to 4.16, while our release-4.15 branch has 4.15 in the same place. Before this PR, I'd expect the master branch to follow at least 4.15, and not 4.14.

> As of now release-4.16 and master is same till the dev freeze happen on openshift side so I am putting respective changes to master and cherry-pick to release-4.16

You said there's a 4.17 microshift branch, but we make changes in snc master so that it follows 4.16. And we have a 4.16 branch which will also follow 4.16. 
And well, we have changes in the 4.15 branch which are not in master ( https://github.com/crc-org/snc/commit/a7c8aac4d03e908c200d725c262c2b493a1b5c63 )

Reply:
> yes because for 4.17 branch there is no package/artifacts available publicly to do testing on the CI. But if you look at openshift/microshift or openshift/installer the main branch in sync with release-4.17 and release-4.16.

Not sure it is really useful for us to have a 4.16 branch, and master implicitly tracking 4.17 but actually also being 4.16. More cherry-picking to do which we can forget.